### ISSUE TITLE
integrate VDJServer's visualizations into the shared visualization library

### ISSUE THREAD

**Issue by schristley** [ireceptor-plus/issues#86]
Hey @marcoamarooliveira @ajrocha @bcorrie , I'd like to start incorporating some of VDJServer's advanced visualizations into the [visualization library](https://github.com/ireceptorplus-inesctec/airrvisualizationlibrary), and then start using it for VDJServer itself. First question though, is that library going to be an "official" iR+ product? Meaning, is it being used for the visualizations in the Gateway, going to extended, supported, etc.?

**Comment by marcoamarooliveira** [comment]
Yes the idea is for the Visualization Library to be an iR+ result/product.

**Comment by schristley** [comment]
Ok, cool, if you give me access to the github repository then I submit PRs for any fixes/enhancements as I integrate.
